servant/src/Servant/API/Verbs.hs

```diff
@@ -21,12 +21,20 @@ import           Network.HTTP.Types.Method
                  methodGet, methodHead, methodOptions, methodPatch, methodPost,
                  methodPut, methodTrace)
 
+import Servant.API.NoContent
+
 -- | @Verb@ is a general type for representing HTTP verbs (a.k.a. methods). For
 -- convenience, type synonyms for each verb with a 200 response code are
 -- provided, but you are free to define your own:
 --
 -- >>> type Post204 contentTypes a = Verb 'POST 204 contentTypes a
-data Verb (method :: k1) (statusCode :: Nat) (contentTypes :: [*]) (a :: *)
+type Verb (method :: k1) (statusCode :: Nat) (contentTypes :: [*]) (a :: *)
```

Comment:
It's very nice that `Verb` can be nearly backwards compatible in this way. But maybe it's `Get`, `Put`, etc that should have this sort of synonym, and we can just redefine `Verb`, since `Verb` is much less user-facing?

Reply:
You are probably right. I don't remember writing `Verb` that often myself.